### DISCUSSION TITLE
How to update src property of an image without triggering update of the whole component?

### DISCUSSION THREAD

**Discussion by marcin-koziol** [zauberzeug/nicegui#1871] [Q&A]
### Question

My goal is to have a live preview of something, images are transferred using websocket.
The problem is, when using 

```python
        screenshot.set_source(f'data:image/png;base64,{img}')
```
it shows a fade-in animation each time the img changes

**Comment by falkoschindler** [comment]
You can use Quasar's "no-transition" and/or "no-spinner" prop, see https://quasar.dev/vue-components/img.

Alternative you can use [`ui.interactive_image`](https://nicegui.io/documentation/interactive_image), which is even more optimized for frequent source changes. It intelligently drops images if they can't be loaded before the next update happens.
